### PR TITLE
Add support for vcpkg

### PR DESCRIPTION
**Pull request by danieljurek**

### CODE REVIEW

File: tools/pipeline-generator/Azure.Sdk.Tools.PipelineGenerator/Conventions/WeeklyIntegrationTestingPipelineConvention.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+        protected override Schedule CreateScheduleFromDefinition(BuildDefinition definition)
+        {
+            var bucket = definition.Id % TotalBuckets;
+            var startHours = bucket / BucketsPerHour;
+            var startMinutes = bucket % BucketsPerHour;
+
+            var schedule = new Schedule
+            {
+                DaysToBuild = ScheduleDays.Saturday | ScheduleDays.Sunday,
```
Comment (mitchdenny) [inline tools/pipeline-generator/Azure.Sdk.Tools.PipelineGenerator/Conventions/WeeklyIntegrationTestingPipelineConvention.cs:25]: Is the only difference in schedule here the days?

**Comment by weshaggard** [comment]
Looks like you have mixed the pipeline generation changes with the vcpkg work. It is probably best to keep those changes separate.

**Comment by weshaggard** [comment]
Looks like the pipeline generation is already part of https://github.com/Azure/azure-sdk-tools/pull/1026.

**Review by mitchdenny — DISMISSED** [review]
Pipeline Generator changes need to be cleaned up.

**Comment by weshaggard** [comment]
Updated PR to remove the pipeline generation commit

File: tools/pipeline-generator/Azure.Sdk.Tools.PipelineGenerator/Conventions/WeeklyIntegrationTestingPipelineConvention.cs (hunk 1 above)
Comment (damirault) [reply tools/pipeline-generator/Azure.Sdk.Tools.PipelineGenerator/Conventions/WeeklyIntegrationTestingPipelineConvention.cs]: This part of the PR has been split out.

**Review by damirault — APPROVED** [review]

**Pull request merged**